src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String FIELD_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final int AMOUNT_OF_SUPPLY_INDEX = 0;
+    private static final int AMOUNT_OF_BUY_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[]amounts = calculateAmounts(fromFileName);
```

Comment:
```suggestion
        int[] amounts = calculateAmounts(fromFileName);
```
use IDEA formatter (Ctrl Alt L) before committing